packages/frontend/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -3064,5 +3064,13 @@
 	"freeAi.credits.showError.claim.title": "Free AI credits",
 	"freeAi.credits.showError.claim.message": "Enable to claim credits",
 	"freeAi.credits.showWarning.workflow.activation.title": "You're using free OpenAI API credits",
-	"freeAi.credits.showWarning.workflow.activation.description": "To make sure your workflow runs smoothly in the future, replace the free OpenAI API credits with your own API key."
+	"freeAi.credits.showWarning.workflow.activation.description": "To make sure your workflow runs smoothly in the future, replace the free OpenAI API credits with your own API key.",
+	"insights.banner.title": "Production executions for the last {count} days",
+	"insights.banner.timeSaved.tooltip": "No estimate available yet. To see potential time savings, {link} to each workflow from workflow settings.",
+	"insights.banner.timeSaved.tooltip.link.text": "add time estimates",
```

Comment:
Does this need a capital (`Add`) or are tooltips different?

Reply:
This is the technique how you add HTML links safely to translations.
You separate them and use `<i18n-t>` tag with templates

If you check [here](https://github.com/n8n-io/n8n/blob/pay-2578-implement-overview-banner/packages/frontend/editor-ui/src/features/insights/components/InsightsSummary.vue#L63) probably it gives you an idea how it works

`<template #link>` means place its content here `{link}` in the translation